packages/skill-api/src/core/services/stripe-checkout.ts

```diff
@@ -408,6 +412,13 @@ export async function stripeCheckout({
         )
       }
     } catch (err: any) {
+      if (errorRedirectUrl) {
+        return {
+          redirect: errorRedirectUrl,
+          status: 303,
+        }
+      }
+
       return {
         status: 500,
```

Comment:
This is where we can redirect to a custom error page for an app instead of just rendering a 500